scripts/generate_pstokes_run.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""
+Pipeline script for generating pstokes visibility.
+"""
+
+from hera_pspec import pstokes
+from pyuvdata import UVData
+import pyuvdata
+
+ap = pstokes.generate_pstokes_argparser()
+args = ap.parse_args()
+uvd = UVData()
+uvd.read(args.inputdata)
+if args.outputdata is None:
+    args.outputdata = args.inputdata
+# generate pstokes and append to file.
+for p in args.pstokes:
+    if pyuvdata.utils.polstr2num(p) not in uvd.polarization_array:
+        uvd += pstokes.construct_pstokes(uvd, uvd, pstokes=p)
```

Comment:
I'm fairly certain that saving data files with both linear and stokes pol can be quite messy for some data types (e.g. fits) that require an even stride between the polarization integers (e.g. -5, -6, -7, -8).

also not clear that it makes sense to write another data file to disk with copies of the vis pol data, just unnecessary data size bloat, no?

Reply:
Agreed. Storing lin-pols in the same file works fine in uvh5 but it causes problems with uvfits. I've added a new `--inplace` arg that is default `False`.